config/initializers/recaptcha.rb

```diff
@@ -0,0 +1,4 @@
+Recaptcha.configure do |config|
+  config.site_key  = ENV['RECAPTCHA_SITE_KEY']
+  config.secret_key = ENV['RECAPTCHA_SECRET_KEY']
```

Comment:
Let's use the settings

```suggestion
  config.site_key = Settings.recaptcha.site_key
  config.secret_key = Settings.recaptcha.secret_key
```